apps/server/src/setup/loadDb.ts

```diff
@@ -21,58 +24,68 @@ const populateDb = (directory: string, filename: string): string => {
   let dbPath = join(directory, filename);
 
   // if everything goes well, the DB in disk is the one loaded
-  // if dbInDisk doesn't exist we want to use startup db
+  // if dbInDisk doesn't exist we create an empty file from db model
   if (!existsSync(dbPath)) {
     try {
-      const dbDirectory = resolveDbDirectory;
-      const startDbName = pathToStartDb.split('/').pop();
-
-      if (!startDbName) {
-        throw new Error('Invalid path to start database');
+      consoleHighlight('No active db found creating empty default');
+      const newFileDirectory = join(resolveDbDirectory, 'new empty project.json');
+      if (!existsSync(newFileDirectory)) {
+        // if it is already there dont override it
+        writeFileSync(newFileDirectory, JSON.stringify(dbModel));
       }
-
-      const newFileDirectory = join(dbDirectory, startDbName);
-
-      copyFileSync(pathToStartDb, newFileDirectory);
+      writeFileSync(appStatePath, JSON.stringify({ lastLoadedProject: 'new empty project.json' }));
       dbPath = newFileDirectory;
     } catch (_) {
       /* we do not handle this */
     }
```

Comment:
question: if we dont catch this, can the app work at all? should we instead just let the error bubble up so the app can crash with an uncaught error?